src/isPxth.ts

```diff
@@ -0,0 +1,4 @@
+import { SegmentsToken } from './getPxthSegments';
+
+export const isPxth = (path: object) =>
+    Boolean((path as { [SegmentsToken]: string[] })[SegmentsToken]);

```

Comment:
This check lacks types. Also, expression could be simplified:
```suggestion
export const isPxth = (path: unknown): path is Pxth<unknown> => SegmentsToken in path;
```